devcsomnicg/iterable-web-sdk
# Dev 249 & 250 (Add embedded views to the SDK & integrate into sample app)

## Description

- **Release Notes**
> Develop different views like card view, banner view, and notification view to display embedded messaging in different views.

> We've updated the sample app to demonstrate the process of displaying embedded messaging across various views.

- **Test Steps**
> Begin by opening the sample app, then choose the 'embedded msgs' option. Enter your user ID in the designated field. Afterward, select the option you wish to test and click. You will be able to see the different views.
